app/models/auditing/UserHasPenaltyAuditModel.scala

```diff
@@ -46,6 +46,9 @@ case class UserHasPenaltyAuditModel(
       ""
   }
 
+  private val lspChargesPaid: Seq[LSPDetails] = penaltyDetails.lateSubmissionPenalty.map(_.details.filter(point => point.chargeOutstandingAmount.contains(0)
```

Comment:
Could you use count instead of filter?

Reply:
And then change the val name to prefix amount

Reply:
Done ✅